players/views.py

```diff
@@ -143,7 +143,7 @@ def watch_level(request, num):
     LOGGER.debug('Displaying game with id %s', game.id)
     return _render_game(request, game)
 
-
+# This function adds the Levels to the database
```

Comment:
might be better as a documentation below the title? But the title is explicit enough I think?

Reply:
:+1: